<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="btu_stu">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>学员信息</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_schedule">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>课程安排</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_finance">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>财务信息</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_honor">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>荣誉排行</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_system">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>系统设置</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>240</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="page_stuInfo">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>260</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_finance">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>200</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_honor">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>290</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_system">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>270</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_schedule">
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>250</y>
          <width>53</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1188</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
